flang/test/Fir/OpenACC/propagate-attr-folding.fir

```diff
@@ -0,0 +1,39 @@
+// RUN: fir-opt %s --opt-bufferization | FileCheck %s 
+
+// Check that OpenACC attributes are propagated to the defining operations when
+// fir.box_addr is folded in bufferization optimization.
```

Comment:
Did you intend to use `fir.box_addr` in the test?  I do not see any :)

Reply:
My bad I pushed the wrong test. I'll update.